tests/getMetadata.test.js

```diff
@@ -68,9 +82,11 @@ describe('Get Metadata Tests', function() {
     };
 
     const metadata = getMetadata(doc, sampleUrl, rules);
+
+    assert.equal(metadata.url, sampleUrl, 'Error finding URL');
     assert.equal(metadata.title, sampleTitle, 'Error finding title');
     assert.equal(metadata.description, sampleDescription, 'Error finding description');
-    assert.equal(Object.keys(metadata).length, 2);
+    assert.equal(Object.keys(metadata).length, 3);
```

Comment:
TIL `chai.assert()` has a `.lengthOf()` method: http://chaijs.com/api/assert/#method_lengthof

Could potentially micro-optimize to:
```js
assert.lengthOf(Object.keys(metadata), 3, 'Has 3 metadata results');
```

I can't seem to find an API that would let us check `metadata` directly instead of having to extract `Object.keys()` into an Array first.